### DISCUSSION TITLE
New blade vanities

### DISCUSSION THREAD

**Discussion by cloudziq** [redeclipse/base#1120] [Maps and Content Showcase]
I remade a blade and blade/twin vanities, old ones are poorly made, using very low resolution for textures, these are badly positioned on player models too

All of three vanities presented here are finished and ready for new release (2.0)

This is how old one blade looks (on left) compared to my rework (on right):
<a href="https://imgur.com/lZfIHrO"><img src="https://i.imgur.com/lZfIHrOl.jpg" title="source: imgur.com" /></a>

a closer view to grips:
<a href="https://imgur.com/9VyIKfR"><img src="https://i.imgur.com/9VyIKfRl.jpg" title="source: imgur.com" /></a>

two more screens:
<a href="https://imgur.com/1mCrXOP"><img src="https://i.imgur.com/1mCrXOPl.jpg" title="source: imgur.com" /></a>
<a href="https://imgur.com/UixTpLz"><img src="https://i.imgur.com/UixTpLzl.jpg" title="source: imgur.com" /></a>

Also I made a totally new one item, a 'modules':
<a href="https://imgur.com/14oIk68"><img src="https://i.imgur.com/14oIk68l.jpg" title="source: imgur.com" /></a> 

Anyway I have a question regarding blade and blade/twin. I want to PR this into master and since I don't want to decide by myself about deleting old blades entirelly, I need opinions about that. If you ask me, there is no point in keeping old, when better one is available, but that just me.

**Comment by ZeroKnight** [comment]
Why is the grip as wide as the blade? It looks really odd.

**Comment by cloudziq** [comment]
I can change this, nobody complained so far ;)    should be a little thinner ?

**Comment by ZeroKnight** [comment]
At *least* half as thin, with some form of guard (you could get pretty creative thanks to the sci-fi theme). As it is now, unless you have hands the size of Shaq, just holding that thing, let alone swinging it would be very unwieldy.

**Comment by cloudziq** [comment]
its just for decoration but as you wish, going to change it now, suggestions are always welcome.

EDIT:  I changed this a little, but in lazy way, sorry, I don't want to create any textures for this from scratch.
Also these models are presented a half of year ago.. so a little late ;)    My goal was to create a better ones, with proper positioning..
this is how grip looks now, hope its better than thicker one: 
<a href="https://imgur.com/EFMDeHX"><img src="https://i.imgur.com/EFMDeHXl.jpg" title="source: imgur.com" /></a>

**Comment by shacknetisp** [comment]
I agree with @ZeroKnight on the grip size, but otherwise these look good!

Since they are direct replacements of the old blade vanities, I think you should overwrite the old vanities with your new versions when you PR them.

**Comment by ZeroKnight** [comment]
Definitely an improvement. 👍 I still feel that they could be thinner, but that's just me.

**Comment by cloudziq** [comment]
its a  futuristic grip with special soft material on it, that fits any hand ideally ;)
